lib/setup.py

```diff
@@ -59,9 +56,13 @@
 ]
 
 # We want to exclude some dependencies in our internal conda distribution of
-# Streamlit.
+# Streamlit. These dependencies will be installed normally for both regular conda builds
+# and PyPI builds.
 CONDA_OPTIONAL_DEPENDENCIES = [
     "gitpython!=3.1.19",
+    "pydeck>=0.1.dev5",
+    # 5.0 has a fix for etag header: https://github.com/tornadoweb/tornado/issues/2262
+    "tornado>=5.0",
```

Comment:
Will moving tornado here break vanilla installations for conda users?

Reply:
(Assuming the answer is no, can we add a comment to this block explaining that `conda install streamlit` will still result in an installation that works correctly?)

Reply:
Yeah, the answer is that this won't affect normal conda users. I'll add an explanatory comment here in this PR and also add an item to my TODO list to make it more obvious that this stuff is for Snowpark-internal reasons only.